packages/flutter_tools/lib/src/localizations/localizations_utils.dart

```diff
@@ -355,7 +355,7 @@ class LocalizationOptions {
     bool? useEscaping,
     bool? suppressWarnings,
     bool? relaxSyntax,
-  }) : templateArbFile = templateArbFile ?? 'app_en.arb',
+  }) : templateLocale = templateLocale ?? 'en',
```

Comment:
will this not default existing users with a template-arb-file field in their l10n.yml to `en` locale?